hypertrace-gradle-docker-plugin/CHANGELOG.md

```diff
@@ -17,4 +17,9 @@ tag using the `CIRCLE_BRANCH` env variable if defined, else `test`
 
 ## [0.6.1]
 ### Fixed
-- Disregard empty values for `CIRCLE_BRANCH` and only use branch name for non-release versions
\ No newline at end of file
+- Disregard empty values for `CIRCLE_BRANCH` and only use branch name for non-release versions
+
+## [0.8.0]
+### Changed
+- Make `COMMIT_SHA` available as a build arg if the SHA can be determined from the environment.
+This currently uses the variable `CIRCLE_SHA1` but further resolution may be added in the future.
```

Comment:
Question: does this mean we are coupled to Circle CI?

Reply:
This plugin is (and already was) - only for fetching git information from environment variables. The alternative is bringing in something like jgit, or using exec directly to run git commands/parse responses - either is doable, but figured we could defer introducing that complexity until needed. Changing that should be transparent to the user.